### PR TITLE
Improves Writing and Removes References to "Functionalities".

### PR DESCRIPTION
**Pull request by Penguin-Guru**
I am not sure if there is a concept of "functionalities" that is different from applications but, since I have not happened across that information and since this is the "community-apps" repo, I have taken the liberty of removing those references. Feel free to leave them in if they do refer to something other than applications, but in that case it might be good to (1) add some documentation to clarify what "functionalities" are and (2) rename this repo to something that does not specify "apps".

### CODE REVIEW

**Comment by Penguin-Guru** [comment]
Ok, I agree that just calling them all apps is fine for now. This might be something to take note of for later though, since re-categorising the apps will be more work when there are more of them. Whoever handles the pull requests can decide. :p

File: web/index.html [hunk 1]
```diff
@@ -17,12 +17,12 @@
 		<div id="wrapper">
 			<div id="content">
 				<h1>Vircadia Community Applications Repository</h1>
-				<p>This repository stores optional <b>tablet applications</b> and <b>functionalities</b> provided by the contributions of the community members. 
-				If you have created an Application or functionality script and want to offer it to the community, this is where you can do it.</p>
+				<p>This repository stores optional <b>tablet applications</b> provided by the contributions of community members who are not core developers of Vircadia.
```
Comment (JulianGro) [inline web/index.html:20]:
```suggestion
				<p>This repository stores optional <b>tablet applications</b> provided by the contributions of community members. 
```
Core developer or not, doesn't play any role here. For example, Davids radar app is in here as well.

File: web/index.html [hunk 2]
```diff
@@ -17,12 +17,12 @@
 		<div id="wrapper">
 			<div id="content">
 				<h1>Vircadia Community Applications Repository</h1>
-				<p>This repository stores optional <b>tablet applications</b> and <b>functionalities</b> provided by the contributions of the community members. 
-				If you have created an Application or functionality script and want to offer it to the community, this is where you can do it.</p>
+				<p>This repository stores optional <b>tablet applications</b> provided by the contributions of community members who are not core developers of Vircadia. 
+				If you have created a tablet application and want to offer it to the community, this is where you can do it.</p>
```
Comment (JulianGro) [inline web/index.html:21]:
```suggestion
				If you have created an application and want to offer it to the community, this is where you can do it.</p>
```
Like talked about; You could have any kind of script in here. Even without a dedicated tablet app button.
I *think* Basinskys VRGrabScale app doesn't use a tablet icon, but I am not sure.

**Review by JulianGro — CHANGES_REQUESTED** [review]

File: web/index.html (hunk 2 above)
Comment (Penguin-Guru) [reply web/index.html:21]: I kind of like the clarity of "tablet application", just so people wouldn't wonder if they can upload applications they have written for other platforms. Either way is good though. :)

File: web/index.html [hunk 3]
```diff
@@ -32,10 +32,10 @@ <h2>Conditions for acceptance:</h2><ol>
...
 
-				<li>Add your application/functionality in a <b>folder</b> under the "<b><a href = 'https://github.com/kasenvr/community-apps/tree/master/applications'>applications</a></b>" directory.<br>
+				<li>Add your application in a <b>folder</b> under the "<b><a href = 'https://github.com/kasenvr/community-apps/tree/master/applications'>applications</a></b>" directory.<br>
 				<font class = 'greencommented' >(Your main js file and your icon must be directly in that folder)</font></li><br>
 
-				<li>You must add your application/functionality to the "<b>metadata.js</b>" file (in the "<b><a href = 'https://github.com/kasenvr/community-apps/tree/master/applications'>applications</a></b>" directory) by generating a new one using this tool: <b><a href='metadata_js_generator.html'>Metadata Generator</a></b></li>
+				<li>You must add your application to the "<b>metadata.js</b>" file (in the "<b><a href = 'https://github.com/kasenvr/community-apps/tree/master/applications'>applications</a></b>" directory) by generating a new one using this tool: <b><a href='metadata_js_generator.html'>Metadata Generator</a></b></li>
```
Comment (digisomni) [inline web/index.html:38]:
```suggestion
				<li>You must add your application to the "<b>metadata.js</b>" file (in the "<b><a href = 'https://github.com/vircadia/community-apps/tree/master/applications'>applications</a></b>" directory) by generating a new one using this tool: <b><a href='metadata_js_generator.html'>Metadata Generator</a></b></li>
```

File: web/index.html [hunk 4]
```diff
@@ -32,10 +32,10 @@ <h2>Conditions for acceptance:</h2><ol>
 				</li><br>
 				<li>Avoid <b>spaces</b> and <b>special characters</b> in your <b>folders</b> and <b>file names</b>.<br><font class = 'greencommented' >(Stay with: "<b>a-z</b>", "<b>A-Z</b>", "<b>0-9</b>", "<b>-</b>", "<b>_</b>", "<b>.</b>")</font></li><br>
 
-				<li>Add your application/functionality in a <b>folder</b> under the "<b><a href = 'https://github.com/kasenvr/community-apps/tree/master/applications'>applications</a></b>" directory.<br>
+				<li>Add your application in a <b>folder</b> under the "<b><a href = 'https://github.com/kasenvr/community-apps/tree/master/applications'>applications</a></b>" directory.<br>
```
Comment (digisomni) [inline web/index.html:35]:
```suggestion
				<li>Add your application in a <b>folder</b> under the "<b><a href = 'https://github.com/vircadia/community-apps/tree/master/applications'>applications</a></b>" directory.<br>
```

**Review by digisomni — CHANGES_REQUESTED** [review]

**Pull request merged**